### PR TITLE
config ID page

### PR DESCRIPTION
**Pull request by jlpoltrack**
Page for Configuration ID

### CODE REVIEW

**Comment by olliw42** [comment]
FANTASTIC

few comments
- there is a bit of a quirk in that it is not possible to have different "crsf","mbridge","in" for different config id's. That is, if one is changing the TxChSource parameter for the current configid, the setting is copied over to ALL configids (and for "in" configid is forced to 0). 
- if a value larger than 9 is selected in EdgeTx/OpenTx, when configid = 9 is used.
- configid is also visible in the OLED, I attach a figure

mlrs-oled-configid-00

File: docs/CONFIGID.md [hunk 1]
```diff
@@ -0,0 +1,19 @@
...
+
+([back to main page](../README.md))
+
+mLRS allows for 10 different configurations to be stored on the Tx module.  This is useful for when you want to use a single Tx module with multiple receivers and want to use different settings for each receiver.  For example, you may prefer to use 50 Hz mode on your copter but prefer to use 19 Hz mode on your rover. The active configuration ID is set based on the receiver number that is assigned on the MODEL SETUP page within the 'External RF' page/section of EdgeTx/OpenTx. This option is only available when using CRSF or mBridge protocols, SBUS will only use configuration ID 0.
+
+## Changing the Receiver Number
+
+Within EdgeTx/OpenTx, navigate to the MDL->MODEL SETUP page and then select the 'External RF' Section:
+
+<img src="images/configIDModelSetup.bmp" width="360px">
```
Comment (olliw42) [inline docs/CONFIGID.md:11]: name does not match file name

**Comment by jlpoltrack** [comment]
@olliw42 
Added in:
- Notes in for the Tx Ch Source parameter
- Notewhen choosing a receiver number greater than 9
- Image showing the. OLED

File: docs/CONFIGID.md (hunk 1 above)
Comment (jlpoltrack) [reply docs/CONFIGID.md]: Good catch - seems the image lookup is case insensitive, the preview was able to load the image fine.  But have fixed the reference to be consistent.

File: docs/CONFIGID.md [hunk 2]
```diff
@@ -0,0 +1,25 @@
+# mLRS Documentation: Configuration ID #
+
+([back to main page](../README.md))
+
+mLRS allows for 10 different configurations to be stored on the Tx module.  This is useful for when you want to use a single Tx module with multiple receivers and want to use different settings for each receiver.  For example, you may prefer to use 50 Hz mode on your copter but prefer to use 19 Hz mode on your rover. The active configuration ID is set based on the receiver number that is assigned on the MODEL SETUP page within the 'External RF' page/section of EdgeTx/OpenTx. This option is only available when using CRSF or mBridge protocols, SBUS will only use configuration ID 0.
+
+Notes:
+- The selected value for the Tx Ch Source parameter will applied to all configuration IDs.
```
Comment (olliw42) [inline docs/CONFIGID.md:8]: just nitpicking
- maybe one should give the values, like "... selected value ("crsf", "in", "mbridge") for ...".
- shouldn't it be "will be applied"

Comment (jlpoltrack) [reply docs/CONFIGID.md]: Updated to address both items.

**Pull request merged**